CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout (BETA)
+  * Fix CocoaPods integrations to pin exact `PayPalCheckout` version `0.100.0`
```

Comment:
Do we want this to be specific to just this version or do something like `Fix CocoaPods integrations to pin exact PayPalCheckout version` since this will also be fixed for all future versions as well?

Reply:
Done!